pkg/inject/sidecar_test.go

```diff
@@ -508,8 +508,9 @@ func TestSidecarOrderOfArguments(t *testing.T) {
 	containsOptionWithPrefix(t, dep.Spec.Template.Spec.Containers[1].Args, "--c-option")
 	containsOptionWithPrefix(t, dep.Spec.Template.Spec.Containers[1].Args, "--jaeger.tags")
 	containsOptionWithPrefix(t, dep.Spec.Template.Spec.Containers[1].Args, "--reporter.grpc.host-port")
-	agentTags := agentTags(dep.Spec.Template.Spec.Containers[1].Args)
-	assert.Contains(t, agentTags, "container.name=only_container")
+	agentTagsMap := parseAgentTags(dep.Spec.Template.Spec.Containers[1].Args)
+	assert.Contains(t, agentTagsMap, "container.name")
+	assert.Equal(t, agentTagsMap["container.name"], "only_container")
```

Comment:
If you are changing this PR for some other reason, you might want to remove the `Contains`, as the `Equals` assertion already covers it.

Reply:
Yes, you're right. Unnecessary `Contains` Removed. Also fix the same issues in other places.